src/AssetPack/AssetPack.router.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from './AssetPack.types'
 import { getDefaultEthAddress } from './utils'
 import { ExpressApp } from '../common/ExpressApp'
+import { sendOk } from 'decentraland-server/dist/server'
```

Comment:
We can delete this import and use `server.sendOk` [here](https://github.com/decentraland/builder-server/pull/285/files#diff-d4ee7ab7527de0ddae7f11f2a1cd1bcfebc1ab0c2550ae5b878b1c9d741e8cc8L1) right?